bananas_server/index/local.py

```diff
@@ -146,8 +146,14 @@ def _read_content_entry_version(self, content_type, unique_id, data, md5sum_mapp
         size += len(md5sum) + 2
         size += len(dependencies) * 4
         size += 1
-        for tag in data.get("tags", []):
-            size += len(tag) + 2
+        for key, value in data.get("classification", {}).items():
+            size += len(key) + 2
```

Comment:
This does not match _send_content_entry

Reply:
Yup. And yes, this is fine ;)

This already checks for the future, that we can always send `{key}:{value}` or `{key}:yes`. The bananas-server implementation also checks if this fits. This way, nobody will introduce a package where we cant send key/value, if we ever want to send the full classification to the client.

In reality, it currently only sends either `key` or `value`, which is always smaller than the combination.

In other words: it does match the validation of bananas-server, and it always sends something smaller as tags.